x-element.js

```diff
@@ -918,9 +918,8 @@ export default class XElement extends HTMLElement {
         computeMap.get(property).valid = false;
       }
       if (queueUpdate) {
-        // Queue a microtask. Allows multiple, synchronous changes.
-        await Promise.resolve();
-        XElement.#updateHost(host);
+        // Batch on microtask to allow multiple, synchronous changes.
```

Comment:
I nearly just deleted the comment because it’s _nearly_ self-documenting. But I think it’s worth highlighting that this is all about _batching_.